src/app/credExplorer/App.js

```diff
@@ -88,6 +104,34 @@ export default class App extends React.Component<Props, State> {
         console.error("Error while fetching:", e);
       });
   }
+
+  analyzePagerankResult(pagerankResult: PagerankResult) {
```

Comment:
I was about to nit about `PageRank` vs `Pagerank`... but realized we already decided to group on semantic capitalization rather than brand capitalization w/ `Github`

Reply:
I was wondering this as I wrote it. My guiding factor was that the identifier `pagerankResult` seems (much) more natural than `pageRankResult`, which suggests that the type name should be `PagerankResult`. Also, consistency, as you note.